README.md

```diff
@@ -85,3 +85,6 @@ After finishing the test, send an email to vagas-dev@juntossomosmais.com.br, wit
 ## Questions
 
 If you have some questions or suggestions, send your questions directly to us (vagas-dev@juntossomosmais.com.br) or opening a issue.
+
+Please, check if your question has already been answered in [a issue](https://github.com/juntossomosmais/frontend-challenge/issues) :relaxed:
```

Comment:
Maybe showing all is better

```suggestion
Please, check if your question has already been answered in [a issue](https://github.com/juntossomosmais/frontend-challenge/issues?q=) :relaxed:
```

Reply:
Done: https://github.com/juntossomosmais/frontend-challenge/pull/4/commits/c89320c649261c0e3d3d8c5b576384ab16b6c6c9